README.md

```diff
@@ -94,6 +94,8 @@ Flags:
       --slack-channel string                slack channel for reboot notfications
       --slack-hook-url string               slack hook URL for reboot notfications
       --slack-username string               slack username for reboot notfications (default "kured")
+      --slack-message-drain string          slack message template when notifying about a node being drained (default "Draining node %s")
```

Comment:
As you suggested, I think it would be better to have it independent of "slack", as many seem eager to have different notification systems (there is even a PR starting the work for that)